doc/howtos/building-images.rst

```diff
@@ -256,6 +284,52 @@ reconfigure it so that it matches ``ostro-image-swupd-dev``::
     OSTRO_IMAGE_NOSWUPD_EXTRA_FEATURES_append = "${OSTRO_IMAGE_FEATURES_DEV}"
     OSTRO_IMAGE_NOSWUPD_EXTRA_INSTALL_append = "${OSTRO_IMAGE_INSTALL_DEV}"
 
+Adding a Custom Layer in Ostro OS
+---------------------------------
+
+There are two practical options for adding a custom layer into your Ostro OS build.
+The first keeps your custom layer independent from ostro-os:::
+
+   $ git clone <your-custom-layer>
+   $ source oe-init-build-env
+   $ bitbake-layers add-layer $LAYERDIR     # add the custom layer path in OSTRO_LAYERS
```

Comment:
I got this off of @mythi 's presentation.  Should I change the OSTRO_LAYERS to BBLAYERS here?